### PR TITLE
make composer.json valid

### LINKED ISSUE #12: composer.json is invalid
Running `composer validate composer.json` you get the following warnings:

* Key require is a duplicate
* License "AGPL" is not a valid SPDX license identifier

### PR DESCRIPTION
**Pull request by garak**
This should fix #12

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -9,13 +9,12 @@
         "ext-xdebug": "*",
         "phpunit/phpunit": "^5.4"
     },
-    "license": "AGPL",
+    "license": "AGPL-1.0",
```
Comment (Lewiscowles1986) [inline composer.json]: It's the AGPL v3 that it's under (in the license file).

File: composer.json [hunk 2]
```diff
@@ -9,13 +9,12 @@
...
     "authors": [
         {
             "name": "Lewis Cowles",
             "email": "lewiscowles@me.com"
         }
     ],
-    "minimum-stability": "stable",
-    "require": {}
+    "minimum-stability": "stable"
```
Comment (Lewiscowles1986) [inline composer.json:19]: I'm assuming this is where the requires was removed, I'd like that left in (it's valid composer & json).

**Comment by Lewiscowles1986** [comment]
Happy to merge once above is fixed. If there is a way to say AGPL 3+, I'd be very happy.

Also if you could create an issue outlining any problems with the current with output, then we can link the issue and the PR, and it makes it look much better for the repo and for your contribution.

Thank you! :smile:

**Review by Lewiscowles1986 — CHANGES_REQUESTED** [review]

File: composer.json (hunk 2 above)
Comment (garak) [reply composer.json:19]: Well, it's not. If you run `composer validate composer.json`, you get "Key require is a duplicate"

**Comment by garak** [comment]
BTW, the only valid [license codes](https://spdx.org/licenses/) for Affero are AGPL-1.0 and AGPL-3.0

**Comment by Lewiscowles1986** [comment]
Please ensure the license reflects the LICENSE document included in the repo :wink:

**Review by Lewiscowles1986 — APPROVED** [review]

**Comment by Lewiscowles1986** [comment]
Gave you a mention in a patch release :wink:

**Pull request merged**